contracts/BatchReveal.sol

```diff
@@ -88,24 +88,125 @@ abstract contract BatchReveal is
     /// @param batchSeed The random number drawn
     event Reveal(uint256 batchNumber, uint256 batchSeed);
 
+    /// @dev Emitted on setRevealBatchSize()
+    /// @param revealBatchSize New reveal batch size
+    event RevealBatchSizeSet(uint256 revealBatchSize);
+
+    /// @dev Emitted on setRevealStartTime()
+    /// @param revealStartTime New reveal start time
+    event RevealStartTimeSet(uint256 revealStartTime);
+
+    /// @dev Emitted on setRevealInterval()
+    /// @param revealInterval New reveal interval
+    event RevealIntervalSet(uint256 revealInterval);
+
+    modifier initialized() {
+        if (!isInitialized()) {
```

Comment:
Can't we check for `collectionSize != 0` here so we don't need `isInitialized()`?

Reply:
yup good suggestion